### DISCUSSION TITLE
consistency problem in DFC

### DISCUSSION THREAD

**Discussion by zhangxiaomei** [DIRACGrid/DIRAC#7937] [Q&A]
This directory has two files only with lfn, but "size -l" shows they are still in JINR-EOS. 
```
FC:/juno/user/j/jinshuzhu/testtar>size -l
directory: /juno/user/j/jinshuzhu/testtar
Logical Size: 1,073,741,824 Files: 2 Directories: 0

   StorageElement Size          Replicas
========================================
 1 JINR-EOS       1,073,741,824        2
   Total          1,073,741,824        2
-----------------------------------------

Query time 0.10 sec
FC:/juno/user/j/jinshuzhu/testtar>ls
test500M_998.file
test500M_999.file
FC:/juno/user/j/jinshuzhu/testtar>replicas test500M_998.file
lfn: /juno/user/j/jinshuzhu/testtar/test500M_998.file
```
I don't know how it happened and how to deal with it.

**Comment by andresailer** [comment]
You can try running rebuild inside `dirac-dms-filecatalog-cli`, but note that that command takes a long time to run if you have many files. (https://github.com/DIRACGrid/DIRAC/issues/2382)

**Reply by zhangxiaomei** [reply]
I have used "rebuild" command to rebuild the table, but failed with timeout every time.  Other methods to fix it?

**Reply by andresailer** [reply]
As far as I remember the command keeps on running in the "background"?
You can figure out how to directly change the table in the file catalog.

**Reply by zhangxiaomei** [reply]
In that case, after "rebuild"  the problem is still there.
It looks like it is the only way to change the table directly.

**Reply by andresailer** [reply]
If I read https://github.com/DIRACGrid/DIRAC/issues/2382 correctly, then you can also try
dirac-dms-filecatalog-cli
```
unregister replica /juno/user/j/jinshuzhu/testtar/test500M_998.file JINR-EOS
unregister replica /juno/user/j/jinshuzhu/testtar/test500M_999.file JINR-EOS
```
